modules/apps/captcha/captcha-api/src/main/java/com/liferay/captcha/simplecaptcha/SimpleCaptchaImpl.java

```diff
@@ -234,6 +222,20 @@ protected BackgroundProducer getBackgroundProducer() {
 		return _backgroundProducers[pos];
 	}
 
+	protected CaptchaConfiguration getCaptchaConfiguration() {
+		if (_captchaConfiguration == null) {
```

Comment:
Hey @daiazius, we can´t cache the configuration since this service won't be notified when the `CaptchaConfiguration` changes anymore. With this change, we may hold a stale version of the CaptchaConfiguration that may cause some bugs. Can you please check that?

cc: @ChrisKian